DB/SQLCreateDataBaseLearnWithMentor.sql

```diff
@@ -1,176 +1,180 @@
---USE master
```

Comment:
1. Why change this file instead of corresponding table files ?
2. Why include this file at all if there are corresponding table files?

Reply:
oh,i forgot making changes in corresponding table files. So, i change them and delete this file

Reply:
remove this file